exchange/exchange.go

```diff
@@ -95,6 +95,10 @@ func NewExchange(client *http.Client, cache prebid_cache_client.Client, cfg *con
 }
 
 func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidRequest, usersyncs IdFetcher, labels pbsmetrics.Labels, account *config.Account, categoriesFetcher *stored_requests.CategoryFetcher, debugLog *DebugLog) (*openrtb.BidResponse, error) {
+	var auc *auction = nil
```

Comment:
There is no need to initialize variables to nil. That's the default. I'm also not sure hoisting the definitions improves readability since it removes the locality.

Reply:
Got it. Will move these declarations back downstream where they are used